packages/box/index.js

```diff
@@ -35,9 +36,13 @@ class WarpBox extends LitElement {
     });
   }
 
+  get _optOutRoleWithDefault() {
+    return this.role === '' ? nothing : this.role ?? 'region'
```

Comment:
I'm not familiar with `nothing`, but if we want to make sure that `role` attribute is hidden when `''` is assigned to it, we could use `ifDefined(this._optOutRoleWithDefault)` directive in line 45 ([lit docs](https://lit.dev/docs/templates/directives/#ifdefined)).

Reply:
https://lit.dev/docs/v1/lit-html/writing-templates/#rendering-nothing

Reply:
Oh, that's cool! Thanks!

Reply:
It gets a bit "messy" when we want a fallback AND still be able to remove the attribute. 😅